### PR TITLE
Fix retrieve command output path

### PR DESCRIPTION
**Pull request by Luni-4**
Fix output path

### CODE REVIEW

File: taskboot/artifacts.py [hunk 1]
```diff
@@ -24,6 +24,6 @@ def retrieve_artifacts(target, args):
     for _, artifact_name, artifact_path in load_named_artifacts(
         config, args.task_id, args.artifacts, args.output_path
     ):
-        logger.info(f"{artifact_name} has been downloaded to {artifact_path}")
+        logger.info("{} has been downloaded to {}".format(artifact_name, artifact_path))
```
Comment (marco-c) [inline taskboot/artifacts.py:27]: Please revert back to using f-string

**Comment by marco-c** [comment]
We should change the arguments of the retrieve command, as the part before the `:` is unused.

**Comment by marco-c** [comment]
> We should change the arguments of the retrieve command, as the part before the `:` is unused.

This way it's confusing for users

**Comment by Luni-4** [comment]
I agree, can we do that in another PR later?

**Comment by marco-c** [comment]
Third time is the charm.

**Comment by Luni-4** [comment]
Hoping the charm doesn't disappear at midnight getting a pumpkin as a result :)

**Pull request merged**